stable/grafana/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: grafana
-version: 5.0.5
+version: 5.0.6
```

Comment:
Should be 5.0.7. Please check.

Reply:
thanks @zanhsieh - I've updated the version